src/web/services/course.service.ts

```diff
@@ -24,4 +24,36 @@ export class CourseService {
     };
     return this.httpRequestService.get('/join', paramMap);
   }
+
+  /**
+   * Send reminder emails to everyone.
```

Comment:
Not to everyone but only to unregistered students. You should change the method name to a proper name also.